src/app/vm-logs/containers/vm-logs.container.ts

```diff
@@ -0,0 +1,37 @@
+import { Component } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { State } from '../../reducers';
+import * as fromVmLogsAutoUpdate from '../redux/vm-logs-auto-update.reducers';
+import * as fromVmLogsVm from '../redux/vm-logs-vm.reducers';
+import * as vmLogsActions from '../redux/vm-logs.actions';
+import { debounceTime } from 'rxjs/internal/operators';
+
+
+@Component({
+  selector: 'cs-vm-logs-container',
+  template: `
+    <cs-vm-logs
+      [isAutoUpdateEnabled]="isAutoUpdateEnabled$ | async"
+      [selectedVmId]="selectedVmId$ | async"
+      (onAutoUpdate)="onAutoUpdate()"
```

Comment:
don't you update this branch?
cause we have a rule for naming here:
https://docs.google.com/document/d/1w_E7JjI60kLOoygIOh-1kzNaMeots_o_IoUCLI4Paxo/edit#